app/routes/logout.tsx

```diff
@@ -0,0 +1,11 @@
+import { ActionFunction } from "@remix-run/node";
+import config from "~/config";
+import { authenticator } from "~/services/auth.server";
+
+export default function Login() {
```

Comment:
I am not sure we need this at all (can't try rn), but if we do it would be Logout and return null

Reply:
I think we do but fixed the name and return